src/main/java/me/dinowernli/grpc/polyglot/config/ConfigurationLoader.java

```diff
@@ -100,49 +99,44 @@ private Configuration getNamedConfigurationInternal(String name) {
 
   /** Returns the {@link Configuration} with overrides, if any, applied to it. */
   private Configuration applyOverrides(Configuration configuration) {
-    if (!overrides.isPresent()) {
+    if (!maybeOverrides.isPresent()) {
       return configuration;
     }
 
+    CommandLineArgs overrides = maybeOverrides.get();
+
     Configuration.Builder resultBuilder = configuration.toBuilder();
-    if (overrides.get().useTls().isPresent()) {
-      resultBuilder.getCallConfigBuilder().setUseTls(overrides.get().useTls().get());
-    }
-    if (overrides.get().outputFilePath().isPresent()) {
+
+    overrides.useTls().ifPresent(resultBuilder.getCallConfigBuilder()::setUseTls);
+
+    overrides.outputFilePath().ifPresent(path -> {
       resultBuilder.getOutputConfigBuilder().setDestination(Destination.FILE);
-      resultBuilder.getOutputConfigBuilder().setFilePath(
-          overrides.get().outputFilePath().get().toString());
-    }
-    if (!overrides.get().additionalProtocIncludes().isEmpty()) {
-      List<String> additionalIncludes = new ArrayList<>();
-      for (Path path : overrides.get().additionalProtocIncludes()) {
-        additionalIncludes.add(path.toString());
-      }
-      resultBuilder.getProtoConfigBuilder().addAllIncludePaths(additionalIncludes);
-    }
-    if (overrides.get().protoDiscoveryRoot().isPresent()) {
-      resultBuilder.getProtoConfigBuilder().setProtoDiscoveryRoot(
-          overrides.get().protoDiscoveryRoot().get().toString());
-    }
-    if (overrides.get().getRpcDeadlineMs().isPresent()) {
-      resultBuilder.getCallConfigBuilder().setDeadlineMs(overrides.get().getRpcDeadlineMs().get());
-    }
-    if (overrides.get().tlsCaCertPath().isPresent()) {
-      resultBuilder.getCallConfigBuilder().setTlsCaCertPath(
-        overrides.get().tlsCaCertPath().get().toString());
-    }
-    if (overrides.get().tlsClientCertPath().isPresent()) {
-      resultBuilder.getCallConfigBuilder().setTlsClientCertPath(
-        overrides.get().tlsClientCertPath().get().toString());
-    }
-    if (overrides.get().tlsClientKeyPath().isPresent()) {
-      resultBuilder.getCallConfigBuilder().setTlsClientKeyPath(
-        overrides.get().tlsClientKeyPath().get().toString());
-    }
-    if (overrides.get().tlsClientOverrideAuthority().isPresent()) {
-      resultBuilder.getCallConfigBuilder().setTlsClientOverrideAuthority(
-          overrides.get().tlsClientOverrideAuthority().get());
-    }
+      resultBuilder.getOutputConfigBuilder().setFilePath(path.toString());
+    });
+
+    resultBuilder.getProtoConfigBuilder().addAllIncludePaths(
```

Comment:
style nit: wrapped lines are indented by 4 (here and below)

Reply:
Do you have an IntelliJ code style I can import? If not, I can make the changes manually.

Reply:
Sadly, not yet. We should definitely add that eventually. For now, manually is the only way i'm afraid.

Reply:
How does this look? https://github.com/grpc-ecosystem/polyglot/pull/67/commits/15e7e57b94e285624add70314ab6d792d462f731